src/Core/Domain/Product/QueryResult/ProductStockInformation.php

```diff
@@ -116,12 +116,12 @@ public function __construct(
         int $outOfStockType,
         int $quantity,
         int $minimalQuantity,
-        string $location,
         int $lowStockThreshold,
         bool $lowStockAlert,
```

Comment:
I think you can rename this field here as well (and especially the getter) with `lowStockAlertEnabled` so that all the commands and query result objects have the same name everywhere (for both Product and Combination)